src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -25,18 +25,18 @@
 //
 //------------------------------------------------------------------------------
 
-using Microsoft.IdentityModel.Logging;
-using Newtonsoft.Json;
 using System;
 using System.Collections.Generic;
+using Microsoft.IdentityModel.Logging;
+using Newtonsoft.Json;
 
-namespace System.IdentityModel.Tokens.Jwt
+namespace Microsoft.IdentityModel.Tokens
 {
     /// <summary>
     /// Represents a Json Web Key as defined in http://tools.ietf.org/html/rfc7517.
     /// </summary>
     [JsonObject]
-    public class JsonWebKey
+    public class JsonWebKey : AsymmetricSecurityKey
```

Comment:
Dunno how I feel about that, still seems a bit artificial to me. Have you considered introducing a `JsonWebKey` -> `SecurityKey` in the JWT package instead?

Also, though symmetric JWK are unlikely to be exposed by a metadata endpoint, developers may still want to use symmetric JWK locally (it's a cool way to serialize a key in a JSON file)